ui/components/multichain/token-list-item/token-list-item.test.tsx

```diff
@@ -66,7 +65,11 @@ describe('TokenListItem', () => {
   it('should render with custom className', () => {
     const store = configureMockStore()(state);
     const { getByTestId } = renderWithProvider(
-      <TokenListItem className="multichain-token-list-item-test" />,
+      <TokenListItem
+        className="multichain-token-list-item-test"
+        tokenImage=""
```

Comment:
Nit Fix: Rather than passing tokenImage and title as empty strings. Can we pass them as props?

Reply:
Sure thing. I'll address this in the downstream PR so that this doesn't get blocked 🙏

Reply:
https://github.com/MetaMask/metamask-extension/pull/26952/commits/6cdd04bd80cd7981551ca27349b868237e2b2bde